README.md

```diff
@@ -88,13 +88,13 @@ The next step is to create an instantiation message for the contract we have:
 
 ```rust
 use sylvia::contract;
-use sylvia::types::ExecCtx;
+use sylvia::types::InstantiateCtx;
 use sylvia::cw_std::{StdResult, Response};
 
 #[contract]
 impl MyContract {
     #[msg(instantiate)]
-    pub fn instantiate(&self, _ctx: ExecCtx) -> StdResult<Response> {
+    pub fn instantiate(&self, _ctx: InstantiateCtx) -> StdResult<Response> {
```

Comment:
BTW - that was working. That is why I am against type aliasing for those types. Also errors are not perfect. We need to reconsider that.

Reply:
I agree. Because in case of compilation error it will point to ExecCtx instead of InstantiateCtx it should be changed to separate types

Reply:
I didn't realize there were different types, as ExecCtx worked here

Reply:
I didn't realize there were different types, as ExecCtx worked here